Mathlib/Data/Option/Defs.lean

```diff
@@ -21,15 +21,6 @@ namespace Option
 
 #align option.lift_or_get Option.liftOrGet
 
-/-- Lifts a relation `α → β → Prop` to a relation `Option α → Option β → Prop` by just adding
-`none ~ none`. -/
-inductive rel (r : α → β → Prop) : Option α → Option β → Prop
-  | /-- If `a ~ b`, then `some a ~ some b` -/
-    some {a b} : r a b → rel r (some a) (some b)
-  | /-- `none ~ none` -/
-    none : rel r none none
-#align option.rel Option.rel
```

Comment:
You should keep this line, but with `Rel` on the RHS.

Reply:
You see, I thought that, but there's another file in Mathlib.Init that *already does this*.

Reply:
Specifically, https://github.com/leanprover-community/mathlib4/blob/master/Mathlib/Init/Data/Option/Basic.lean

Reply:
cc @digama0, I thought #align was supposed to prevent a single lean3 def having multiple lean4 alignments like this.

Reply:
It does, but the check only runs when you write the command, not at `import` time, meaning that if you have two files that don't import each other then you can end up with overlapping alignments.

Reply:
Would you rather have it in here or in the other file?

Reply:
@digama0, is it easy to write a test that checks for duplicates in CI?

Reply:
> Would you rather have it in here or in the other file?

The other file is fine.